### DISCUSSION TITLE
Which microphone ?

### DISCUSSION THREAD

**Discussion by Fish31520** [rdz-oss/BattyBirdNET-Analyzer#2] [Q&A]
Hi guys,

I already have Birdnet-Pi but I would be interested in BattyBirdnet-Pi.

Which (not too expensive 😅) microphone could be used for that purpose ?

Thanks

**Comment by rdz-oss** [comment]
I develop the prototype with wildlife acoustic touch 2
As yet only the ML part is ready. Integration into the Pi system is planned.
Probably the Audiomoth will work - not yet clear

**Reply by Fish31520** [reply]
Thanks !
Well, I'll wait a bit and follow your progress on this very interesting project !
As soon as it will be available on RPi, I'll by the mic and test that.

**Reply by rdz-oss** [reply]
So there is a prototype that is working also with the audiomoth usb micro as well as the echo meter 2.

It is still experimental and currently has the tendency for getting triggered by noise. I will be working to reduce that.
If you only turn it on at dusk when the bats fly you can use it as of now.

``` sh
curl -s https://raw.githubusercontent.com/rdz-oss/BattyBirdNET-Pi/main/newinstaller.sh | bash
```

will get it installed. Work in progress.

**Comment by rdz-oss** [comment]
I ordered an audiomoth 1.2.0 . I will continue developing for the echo meter touch. Yet when that works, I will give the audiomoth a shot, too. Some people have said that the audiomoth has a higher noise level. I will check in time.

**Comment by Xoxoxcu** [comment]
I'd suggest checking out some ASMR microphones. Excellent audio quality while remaining affordable. One popular option you might consider is the Tascam DR-05X Portable Audio Recorder. It's a versatile microphone that offers high-quality sound recording at an affordable price. It has built-in stereo microphones and can also be connected to an external microphone for even better audio capture. If you're looking for a more budget-friendly option, the Fifine K669B USB Microphone is worth considering. Despite its low price, it delivers decent sound quality and is a popular choice among content creators. Remember, when choosing a microphone, it's essential to consider your specific needs and preferences. Btw, here are some reviews to check out https://asmrmicrophones.com/. Look for features like noise cancellation, frequency response, and connectivity options to ensure you find the perfect fit.

**Comment by alexbelgium** [comment]
Hi, based on the faq I see you recommend audiomoth USB https://github.com/rdz-oss/BattyBirdNET/blob/main/FAQ.md#which-microphone-to-use-

Has anyone tried the Clippy XLR ultra ? It seems to have a response up to 110khz at least and is cheaper ?

Edit : actually it seems both clippy XLR and audiomoth use the same MEMS capsule "Knowles Utrasonic Mic SPU0410LR5H-QB" according to [this](https://www.openacousticdevices.info/support/main/comment/f4771d6c-dcce-37ca-bf2a-2aa69346578b?postId=6061b4c1f313fb0015f8bfea) and [this](https://www.airwiggles.com/c/field-recording/clippy-vs-clippy-ultras)

**Reply by rdz-oss** [reply]
Hi @alexbelgium ,
I have not tried that microphone. With 110kHz it would be able to detect calls up to about 55kHz or so, which is enough only for some species. For use in BattyBirdNET-Pi the classifiers would have to be trained on the mics noise profile to avoid many failed detections.
Did you try the Clippy XLR? It would be great to have an alternative. I have tried it with the usb version of https://www.pippyg.com/pipistrelleprev.html and it worked well for detection, the files are very noisy though.

BR
Richard

**Reply by alexbelgium** [reply]
Hi Richard,

Thanks for your feedback. I have not try the Clippy XLR Ultra based on MEMS capsule, but I am actually using the Clippy XLR  based on EM272 capsule with a Focusrite Scarlett 2i2 as audio converter. Graphs are very clean. Specs claim until 20khz, but on my graphs though I see a response until around 60khz

Edit : this is without any sound reduction applied

<img width="300" alt="image" src="https://github.com/user-attachments/assets/3d508e03-4af4-43db-a3b1-274adab905a7" />

<img width="300" alt="image" src="https://github.com/user-attachments/assets/ab80a4ff-b68c-4d92-b23b-435ee4e70200" />

Best regards.

**Reply by rdz-oss** [reply]
Hi Alex,

 looks alright ! If you record birds and some bat species without need for perfection this is a great option (if you also have a Focusrite Scarlett 2i2 ). I got lots of feedback from bat conservationists who want at least 384kHz (about 190kHz signal capture) to get the full range of European bats, they would even prefer 500kHz. 

-> There is a lot of value in getting some bats, which might also be most that fly through your yard. Most people do not require the full spectrum just to enjoy recording some bats.

I would like to try some Petterson and dodotronic mics some time for the larger ranges.

-BR
Richard

**Comment by rdz-oss** [comment]
My current recommendation is the usb audiomoth: https://github.com/rdz-oss/BattyBirdNET-Pi/discussions/34
